### PR TITLE
IOD-103 Navigation added

### PR DESCRIPTION
**Pull request by aideninbcgov**
# Description

This PR includes the following proposed change(s):

- IOD-103

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/common/UserMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+// import { Avatar, Chip, Typography, Box, Menu, MenuItem, ListItemIcon , Logout } from '@mui/material';
```
Comment (akroon3r) [inline src/components/common/UserMenu.jsx:1]: Do we need this comment going forward?

File: src/components/navigation/Navigation.jsx [hunk 2]
```diff
@@ -1,40 +1,52 @@
...
+import slamlogo from 'assets/images/slam-logo.png';
 import Image from 'next/image';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import React from 'react';
+import {React, useState } from 'react';
+import { AvatarChip, UserMenu } from 'components';
+
+
+
```
Comment (akroon3r) [inline src/components/navigation/Navigation.jsx:10]: I'm a stickler for un-needed white space. What's best practice between imports and first line of code? 2 or 3?

**Review by akroon3r — COMMENTED** [review]
Looks good so far, just a couple of questions from me. Lets see what Scott has to say as well.

File: src/components/common/UserMenu.jsx (hunk 1 above)
Comment (aideninbcgov) [reply src/components/common/UserMenu.jsx:1]: Good point. Thanks for catching it. 
Usually when things are from the same library, we can import as single line. 
e.g import {a,b.c} from 'library'
but somehow I got errors (conflict with Next.js), so I googled it, and someone suggested to separate them. I still don't know the rational behind. but it worked.

File: src/components/navigation/Navigation.jsx (hunk 2 above)
Comment (SDToews) [reply src/components/navigation/Navigation.jsx:10]: I usually leave one whitespace line

File: src/components/common/UserMenu.jsx (hunk 1 above)
Comment (SDToews) [reply src/components/common/UserMenu.jsx:1]: The import above is named items, but the ones below are unnamed default values.  Probably the name of the items don't match exactly, which would explain why you were getting an error.  This is common for MUI Icons.

**Review by SDToews — APPROVED** [review]

**Pull request merged**